api/pom.xml

```diff
@@ -13,15 +13,15 @@
 	<description>API for soknadsfillager</description>
 
 	<properties>
-		<spring-web.version>5.3.13</spring-web.version>
+		<spring-web.version>5.3.14</spring-web.version>
```

Comment:
Why the dependency to spring-web in api module. Jeh tror jeg spørte det samme spørsmål på en anne pull request allerede.

Reply:
Det trengs for at koden skal bygge. Jeg testet at fjerne spring-web, men då bygger ikke komponenten.